app/routes/_index.tsx

```diff
@@ -23,6 +27,20 @@ export const meta: V2_MetaFunction = () => {
 
 export default function Index() {
   const tekster = useTekster(ESanitySteg.FORSIDE);
+  const [samtykkeErBekreftet, settSamtykkeErBekreftet] = useState(false);
```

Comment:
```suggestion
  const [erSamtykkeBekreftet, settErSamtykkeBekreftet] = useState(false);
```
Skal vi starte med en konvensjon om at vi plaserer "er" først? Bare for å være konsistente på det ? :)

Reply:
Ja, jeg er hvertfall veldig ombord på det!